### DISCUSSION TITLE
Shop Not Found Error

### DISCUSSION THREAD

**Discussion by prathore90** [kinngh/shopify-node-express-mongodb-app#40] [Q&A]
Thanks for creating this template. I really appreciate your help. I have been following this repository from the last version, which is still working fine. But this new updates throws  `No shop provided` error. I have been following the same instructions provided in the Youtube video, and the old template still works fine, but no luck so far with the new template.

**Comment by kinngh** [comment]
Can you please add in screenshots, video cast, something so I can actually see what's happening?

**Reply by prathore90** [reply]
Thanks for the response. Here is the small video: https://www.loom.com/embed/9c44c2265b494d33bd5fd7bd8439b577

**Reply by kinngh** [reply]
Can you drop in screenshots of your MongoDB table `active_stores` and `session`?

**Reply by prathore90** [reply]
I am using mongodb atlas, and when I try to install the app on a store, it does not generate any doc. The active store and sessions are empty for now.
<img width="1237" alt="image" src="https://user-images.githubusercontent.com/116027221/209296524-3409e663-2fb9-4f38-899a-ba759bb6e20e.png">
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/116027221/209296820-f77d769a-be38-4d24-94c9-cad90920729c.png">

**Reply by prathore90** [reply]
This is the error, it throws when I try to install the app with url
<img width="745" alt="image" src="https://user-images.githubusercontent.com/116027221/209297042-e209d4f3-6933-4111-93f9-33031d9a2a59.png">

**Reply by kinngh** [reply]
Can you DM me on Twitter so we can setup a screen share? This is a clear problem with not being able to connect to MongoDB

**Comment by jaiswalshash** [comment]
Hey @kinngh I am getting the same error. Can you please tell me what is the solution for this?

**Reply by kinngh** [reply]
is your MongoDB connection working? Double check if you have tables being filled up with data. Also, you have given me 0 context on this, so it's impossible for me to help you.

**Comment by jaiswalshash** [comment]
Apologies. So I was facing the same issue as above. Tables are not getting filled 
![image](https://github.com/user-attachments/assets/886d6b74-8b99-417b-a811-05330668d84e)
![image](https://github.com/user-attachments/assets/d90dc3bf-b033-4ecf-9336-25e4ed708fe2)

I'm using this template to build a non-embedded app

**Reply by kinngh** [reply]
Yeah this is a database connection issue. Double check your connection string and see if your database is accessible - there's no other issue that causes this. If it's still a problem, install MongoDB locally and connect locally, which should work because I just pulled the repo and it works just fine^

**Reply by kinngh** [reply]
Also, please reply in thread instead of starting a new one every single time^
